libpod/diff.go

```diff
@@ -29,6 +29,13 @@ func (r *Runtime) getLayerID(id string) (string, error) {
 	var toLayer string
 	toImage, err := r.GetImage(id)
 	if err != nil {
+		if id == "**latest" {
+			ctr, err := r.LookupContainer(id)
+			if err != nil {
+				return "", errors.Wrapf(err, "unable to look up latest container")
+			}
+			id = ctr.ID()
+		}
 		toCtr, err := r.store.Container(id)
```

Comment:
This also really need to be converted to use LookupContainer instead of r.store.Container - can you drop a TODO here too?

Reply:
ok